compressors/prefix_free_compressors.py

```diff
@@ -134,16 +135,62 @@ def print_tree(self):
     def build_tree(prob_dist) -> PrefixFreeTreeNode:
         """
         abstract function -> needs to be implemented by the subclassing class
+
+        needs to build the whole tree and return the pointer to root node
         """
         raise NotImplementedError
 
+    def build_tree_from_code(self, symbol, code) -> PrefixFreeTreeNode:
```

Comment:
Consider making this function a "private function" by renaming it with `_build...` ..
Also, maybe rename this: `_add_code_to_the_tree` or something similar which might be more appropriate

Reply:
Also complete the docstring!